aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -45,7 +47,8 @@ c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>> PackedConvWeight<
       " elements for ",
       kSpatialDim,
       "D convolution.");
-  const int output_channels = weight.size(0);
+  const int output_channels_idx = transpose ? 1 : 0;
```

Comment:
optional: if you want these to be more readable, you could define an enum `ConvType` which can be `ConvType::CONV | ConvType::CONV_TRANSPOSE` and pass that around instead

Reply:
I love this solution -- I was originally thinking of casting boolean to int (as in `int(transpose)`), but thought that would be confusing when used as a channel index